test/ui/ui-accuracy.pool.investing.test.ts

```diff
@@ -57,24 +58,39 @@ describe("UI tests - Get Tokens from Faucet", () => {
     sudo = new User(keyring, sudoUserName);
     await sudo.mint(new BN(0), testUser1, new BN(visibleValueNumber));
   });
-  beforeEach(async () => {
-    [firstCurrency, secondCurrency] = await Assets.setupUserWithCurrencies(
-      testUser1,
-      [new BN(visibleValueNumber), new BN(visibleValueNumber)],
-      sudo
-    );
-  });
-  ///TODO: Extend when locators available.Right now I can not select more liq. pools- I dont have Ids, and I dont have asset names :S
   test.each([
     [
-      new BN(Math.pow(10, 19).toString()).div(new BN(2)),
-      new BN(Math.pow(10, 19).toString()).div(new BN(2)),
+      new BN(Math.pow(10, 19).toString()).div(new BN(2)), //5E19X
+      new BN(Math.pow(10, 19).toString()).div(
+        new BN(Math.pow(10, 17).toString()) //100Y
+      ), // there is a difference of 17 decimals. Pool[ 5E19 X - 100Y]
+      "100000000000",
+      "0.000002",
+    ],
+    [
+      new BN(Math.pow(10, 19).toString()).div(new BN(2)), //5X
+      new BN(Math.pow(10, 19).toString()).div(new BN(2)), //5Y
       "0.123",
       "0.123",
     ],
+    [
+      new BN(Math.pow(10, 19).toString()).div(new BN(4)), //2.5X
+      new BN(Math.pow(10, 19).toString()).div(new BN(2)), //5Y
+      "0.100",
+      "0.200",
+    ],
   ])(
```

Comment:
@PetoU 
I remember that you were interested in some accuracy tests that validate that the UI correctly calculates the entered amounts.
This test is a data-driven test that validates 
[
 NumberOfAssetX
 NumberOfAssetY
 Amount to enter
 Expected
]
 feel free to ping me if you are interested in testing any amount for minting liquidity calculations. 
Right now I am testing 3 equivalence partitioning use cases.
With an unfair pool, 
* with 5E19 - 100
With a perfect balance 
* 5E19, 5E19
With a 1X -2y